app/assets/stylesheets/application.scss

```diff
@@ -3,9 +3,12 @@
 @import url('https://fonts.googleapis.com/css2?family=Indie+Flower&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Titillium+Web:wght@700&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Anonymous+Pro&family=Yantramanav&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Archivo:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Modak&display=swap');
```

Comment:
You can consolidate all of your google font imports.  Not a deal breaker, but would be a good cleanup of your scss file.  Google allows you to add multiple fonts when selecting, and will give you one import to rule them all.